dist/PathTools/Cwd.pm

```diff
@@ -192,8 +192,14 @@ sub _backtick_pwd {
     # Localize %ENV entries in a way that won't create new hash keys.
     # Under AmigaOS we don't want to localize as it stops perl from
     # finding 'sh' in the PATH.
-    my @localize = grep exists $ENV{$_}, qw(PATH IFS CDPATH ENV BASH_ENV) if $^O ne "amigaos";
+    my @localize = grep exists $ENV{$_}, qw(IFS CDPATH ENV BASH_ENV) if $^O ne "amigaos";
     local @ENV{@localize} if @localize;
+    # empty PATH is the same as "." on *nix, so localize it to /something/
+    # we won't *use* the path as code above turns $pwd_cmd into a specific
+    # executable, but it will blow up anyway under taint. We could set it to
+    # anything absolute. Perhaps "/" would be better.
+    local $ENV{PATH}= "/usr/bin"
+        if $^O ne "vms" and $^O ne "amigaos";
```

Comment:
> This patch does not (or rather should not) change behavior for VMS.

@demerphq Note that this _is_ a change of behaviour under VMS: we no longer localize `$ENV{PATH}` to undef, but we also don't localize it to `'/usr/bin'`.

I think the change to line 195 is not needed, and should probably be undone.

(Sorry I didn't think to look at this until after it was merged.)

Reply:
@hvds: see https://github.com/Perl/perl5/pull/20576